assets/css/style.css

```diff
@@ -0,0 +1,153 @@
+@media (min-width: 375px) {
+  .container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .header-menu {
+    display: flex;
+    flex-direction: row-reverse;
+    margin-top: 5%;
+    height: 10%;
+  }
+
+  #hamburg-icon {
+    font-size: 17pt;
+    padding-right: 2rem;
+    color: #28352f;
+  }
+
+  .headline-section {
+    background-image: url(../img/image.png);
+    background-size: 100%;
+    background-repeat: no-repeat;
+    height: 100vh;
+    margin-top: 5%;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-top: 10%;
+  }
+
+  .socialmedia-links {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 10%;
+    margin-top: -2%;
+  }
+
+  .socialmedia-links li { list-style: none; }
+
+  .socialmedia-links img {
+    height: 80%;
+    width: 80%;
+  }
+
+  .headline-primary,
+  .headline-secondary {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+  }
+
+  .headline-primary { color: #172b4d; }
+
+  .headline-secondary { color: #36b37e; }
+
+  .headline-content p {
+    margin-top: -10%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #172b4d;
+    padding-left: 1em;
+    padding-right: 1em;
+  }
+}
+
+@media (min-width: 375px) {
+  .container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .header-menu {
+    display: flex;
+    flex-direction: row-reverse;
+    margin-top: 5%;
+    height: 10%;
+  }
+
+  #hamburg-icon {
+    font-size: 17pt;
+    padding-right: 2rem;
+    color: #28352f;
+  }
+
+  .headline-section {
+    background-image: url(../img/image.png);
+    background-size: 100%;
+    background-repeat: no-repeat;
+    height: 100vh;
+    margin-top: 5%;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-top: 10%;
+  }
+
+  .socialmedia-links {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 10%;
+    margin-top: -2%;
+  }
```

Comment:
- [ ] [Required] Good Job in implementing the changes :clap: , However, As requested by the previous reviewer, Please ensure that your design matches the given Design. When your design matches the given design, it ensures that all elements across the website or application follow a consistent style, such as colors, typography, spacing, and layout. Consistency helps users navigate and interact with the interface more easily, as they become familiar with the visual patterns and can predict how things will work. :slightly_smiling_face: 
- The table below shows the difference between the social Icons, Please fix the Gap: :arrow_down: 
<table>
    <tr>
      <td>Your design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Your Design](https://github.com/Graciano1997/portofolio-graciano/assets/43528594/fd8d81fc-e83a-48c2-9b1a-555af97ef506)

</td>
      <td>

![Social Icons](https://github.com/Graciano1997/portofolio-graciano/assets/43528594/3935321f-e22b-4582-9beb-501a4a1455de)
</td>
    </tr>
  </table>

Reply:
Hello Can I change the template ???

Reply:
what means that black area in my design??